epictrack-api/src/api/resources/work.py

```diff
@@ -74,6 +75,8 @@ def post():
         request_json = req.WorkBodyParameterSchema().load(API.payload)
         request_json["start_date"] = get_start_of_day(request_json["start_date"])
         work = WorkService.create_work(request_json)
+        role_id = CodeService.find_code_values_by_type("roles", { "name": "Team Lead" }).get("codes")[0].get("id")
```

Comment:
Can we move this logic to inside create_work function in work.py service. Cz then it will be in a single transaction level with respect to database.